tests/unit/list/dlist.c

```diff
@@ -173,10 +173,10 @@ static inline bool verify_tail_head(sys_dlist_t *list,
  * sys_dlist_remove(), sys_dlist_insert(), sys_dlist_peek_next()
  * SYS_DLIST_ITERATE_FROM_NODE()
  */
-void test_dlist(void)
+ZTEST(dlist_api, test_dlist)
 {
 	sys_dlist_init(&test_list);
-
+	printf("code is running\n");
```

Comment:
Whoops, looks like we left a print statement here.

Reply:
Sorry, I have removed it. Thanks.